CHANGELOG.md

```diff
@@ -65,6 +65,7 @@ You can find its changes [documented below](#070---2021-01-01).
 - Update look and feel of controls when disabled ([#1717] by [@xarvic])
 - Change the signature of `add_idle_callback` ([#1787] by [@jneem])
 - Move macOS only function to Mac extension trait ([#1863] by [@Maan2003])
+- x11: Only query atoms once instead of per window ([#1865] by [@psychon])
```

Comment:
I am not sure if document internal changes in changelog. @cmyr ?

Reply:
I'm not convinced by this changelog entry either, but: https://github.com/linebender/druid/blob/master/CONTRIBUTING.md#changelog

> Every pull request should document all changes made in the changelog.